pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/V3DefaultColumnHandler.java

```diff
@@ -62,16 +62,28 @@ protected boolean updateDefaultColumn(String column, DefaultColumnAction action)
     FieldSpec fieldSpec = _schema.getFieldSpecFor(column);
     Preconditions.checkNotNull(fieldSpec);
     boolean isSingleValue = fieldSpec.isSingleValueField();
-    File forwardIndexFile;
+    boolean forwardIndexDisabled = !isSingleValue && isForwardIndexDisabled(column);
```

Comment:
Do you remember our discussion with Jackie, where we decided that for SV columns, we allow the forward index to be created as it'll be sorted. This will provide consistent behavior with normal segment creation code path where for sorted columns the `forwardIndexDisabled` flag is essentially a no-op? MV columns can never be sorted so for default column handling we need to handling this differently than SV columns. Default SV columns will always be sorted and thus will always have a forward index.

Reply:
Ohh yes. I remember now. Makes sense.